src/components/settings-menu/index.tsx

```diff
@@ -66,39 +85,48 @@ export class SettingsMenu extends React.Component<Properties, State> {
 
   render() {
     return (
-      <DropdownMenu
-        menuClassName={'settings-menu'}
-        items={[
-          {
-            id: 'header',
-            label: this.renderSettingsHeader(),
-            onSelect: () => {},
-          },
-          {
-            className: 'divider',
-            id: 'divider',
-            label: <div />,
-            onSelect: () => {},
-          },
-          {
-            className: 'logout',
-            id: 'logout',
-            label: this.renderSettingsOption(<IconLogOut3 />, 'Log Out'),
-            onSelect: () => this.handleLogout(),
-          },
-        ]}
-        side='right'
-        alignMenu='start'
-        onOpenChange={this.handleOpenChange}
-        trigger={
-          <Avatar
-            isActive={this.state.isDropdownOpen}
-            size={'medium'}
-            type={'circle'}
-            imageURL={this.props.userAvatarUrl}
-          />
-        }
-      />
+      <div>
+        <DropdownMenu
+          menuClassName={'settings-menu'}
+          items={[
+            {
+              id: 'header',
+              label: this.renderSettingsHeader(),
+              onSelect: () => {},
+            },
+            {
+              className: 'edit_profile',
+              id: 'edit_profile',
+              label: this.renderSettingsOption(<IconUser1 />, 'Edit Profile'),
+              onSelect: () => this.openEditProfileDialog(),
```

Comment:
```suggestion
              onSelect: this.openEditProfileDialog,
```

No need for the lambda. Might as well just pass the reference to the function.